README.md

```diff
@@ -105,8 +105,25 @@ Yamashita | Yah-MAH-sh-Ta  | some vowels (i, u) get removed (called devoicing),
 Kyoko     | Kyoh-koh       | (not Kee-oh-koh)      
 Ryousuke  | Ryoh-skeh      | the -suke suffix is usually skeh
 
+## Portuguese (Brazilian)
+
+The challenge with Portuguese is that there are some phonems (such as \[ɐ̃w̃\], \[s̻\] or \[ʎ\]) and [diacritics](https://en.wikipedia.org/wiki/Diacritic) (such as an accent or cedilla) not common or not at all present in the English language. 
+
+First, a few general rules:
+1) **ão** is a nasal diphthong and resembles "pawn" + "o"
+2) **ç** is like a soft S sound (think of how you'd prounounce the French word [facade](https://en.wiktionary.org/wiki/facade))
```

Comment:
i think the consonant in facade is pronounced the same in english and french (so says wikitionary) so can probably just say facade here.